src/testutil-new-tab.html

```diff
@@ -1,12 +1,17 @@
 <html>
   <head>
     <script type="application/javascript">
```

Comment:
Remove type attribute. JS is default

Reply:
maybe even use `type="module"`which opts you into strict module mode.

Reply:
Thanks, done.